plugins/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/requirement/OpenShiftCommandLineToolsRequirement.java

```diff
@@ -56,12 +61,21 @@ public boolean canFulfill() {
 
 	@Override
 	public void fulfill() {
-		File ocBinary = new File("binaries" + SEPARATOR + "oc");
-		if (!ocBinary.exists()) {
-			downloadAndExtractOpenShiftClient();
+		if (!OCBinaryFile.get().getFile().exists()) {
+			File downloadedOCBinary = downloadAndExtractOpenShiftClient();
+			createSymlink(downloadedOCBinary);
 		} else {
 			logger.info("Binary is already downloaded.");
-			PATH_TO_CLIENT = ocBinary.getAbsolutePath(); 
+		}
+	}
+
+	private void createSymlink(File downloadedOCBinary) {
+		try {
+			Files.deleteIfExists(OCBinaryFile.get().getFile().toPath());
+			Files.createSymbolicLink(OCBinaryFile.get().getFile().toPath(), Paths.get(downloadedOCBinary.getAbsolutePath()));
+		} catch (IOException e) {
+			throw new OpenShiftToolsException(NLS.bind("Could not symlink {0} to {1}:\n{3}", 
```

Comment:
The {3} will be resolved as null, because there are only 3 arguments. There should be {2} instead of {3}.